geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/RedisList.java

```diff
@@ -204,6 +229,32 @@ public int getDSFID() {
     return REDIS_LIST_ID;
   }
 
+  public synchronized int elementInsert(byte[] elementToInsert, byte[] referenceElement,
+      boolean before) {
+    int i = 0;
+    ListIterator<byte[]> iterator = elementList.listIterator();
+
+    while (iterator.hasNext()) {
+      if (Arrays.equals(iterator.next(), referenceElement)) {
+        if (!before) {
```

Comment:
Really small nitpick, but could this if statement be inverted, so that we have `if (before) {` at the top? I find it makes it easier to read if we can avoid negation in the conditions.